continuously_train_user_model.py

```diff
@@ -17,46 +17,34 @@
 import configs
 from adaptive_hci.controllers import PLModel
 from adaptive_hci.datasets import EMGWindowsAdaptationDataset, \
-                                  get_concatenated_user_episodes, \
-                                  load_online_episodes
+    get_concatenated_user_episodes, \
+    load_online_episodes
+
 
 def main(finetuned_model: LightningModule, user_hash, config: configs.BaseConfig) -> LightningModule:
-    _ = wandb.init()
+    maybe_download_drive_folder()
+    _ = wandb.init()  # TODO: do we need to reinit in the pipeline case? if not, we should init wandb outside
```

Comment:
we need to re-init in the case of a sweep, I guess this could be done in the sweep_wrapper function